packages/website/docs/getting-started/theming/utilities/preview_utilities.tsx

```diff
@@ -6,11 +6,19 @@ import {
   useEuiTheme,
   EuiTextAlign,
   EuiFlexGrid,
+  EuiFlexGroup,
   EuiFlexItem,
   EuiPanel,
   EuiTextColor,
   EuiCode,
+  EuiText,
+  EuiSpacer,
 } from '@elastic/eui';
+import {
+  euiFontSizeFromScale,
+  euiLineHeightFromBaseline,
+  euiTextShift,
+} from '@elastic/eui/src/global_styling';
```

Comment:
```suggestion
} from '@elastic/eui';
```

to make the update easier :)